python/risk_meters/list_risk_scores.py

```diff
@@ -0,0 +1,99 @@
+# List Risk Score and True Risks scores.  For each risk meter that the scores
+# are not equal, spawn a process to display the vulnerability counts and risk
+# accepted vulnerability counts.
+
+import os
+import sys
+import subprocess
+import requests
+from prettytable import PrettyTable
+
+# Get risk meter information, and return a dictionary of tuples containing risk meter ID,
+# risk meter score, and true risk meter score.
+def get_risk_meter_scores(base_url, headers):
+    risk_meters = {}
+    list_risk_meters_url = f"{base_url}asset_groups"
+
+    response = requests.get(list_risk_meters_url, headers=headers)
+    if response.status_code != 200:
+        print(f"List Risk Meters Error: {response.status_code} with {list_risk_meters_url}")
+        sys.exit(1)
+
+    resp_json = response.json()
+    risk_meters_resp = resp_json['asset_groups']
+
+    for risk_meter in risk_meters_resp:
+        risk_meter_id = risk_meter['id']
+        #query_string = risk_meter['querystring']
+        risk_meter_score = risk_meter['risk_meter_score']
+        true_score = risk_meter['true_risk_meter_score']
+        risk_meters[risk_meter['name']] = (risk_meter_id, risk_meter_score, true_score)
+
+    return risk_meters
+
+# Obtain and return the number of assets in a risk meter.
+if __name__ == "__main__":
+    print("")
+    print("List Risk Meter Scores")
+    print("~~~~~~~~~~~~~~~~~~~~~~")
+    print("")
+
+    # Obtain the Kenna Security API key from an environment variable.
+    api_key = os.getenv('KENNA_API_KEY')
+    if api_key is None:
+        print("API key is non-existent")
```

Comment:
I'd make this more user-centric like, "In order to run this script, please assign your Kenna Risk Token to the KENNA_API_KEY environment variable."